packages/wp-story-editor/src/components/documentPane/publish/author.js

```diff
@@ -55,10 +55,11 @@ function Author() {
   const [visibleOptions, setVisibleOptions] = useState(null);
 
   useEffect(() => {
-    if (tab === 'document') {
-      loadUsers();
+    if (users.length === 0 && !isUsersLoading) {
```

Comment:
`loadUsers` already includes that check as well: https://github.com/GoogleForCreators/web-stories-wp/blob/d530f151e0afc617b2ce9097d48dd1f5c77411f5/packages/story-editor/src/components/inspector/inspectorProvider.js#L111-L114